.github/workflows/main.yml

```diff
@@ -3,29 +3,45 @@ jobs:
   build-and-deploy:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
-    - name: Install dependencies
-      run: |
-        sudo apt-get install -y rpm
-        sudo gem install fpm package_cloud
-    - name: Build
-      run: |
-        make deb
-        make rpm
-    - name: Deploy
-      if: ${{ github.ref == 'refs/heads/main' }}
-      run: |
-        PACKAGECLOUD_REPO="cs50/repo"
+      - uses: actions/checkout@v2.1.0
```

Comment:
Do we need to fixate on 2.1.0 or does v2 suffice? Or v2.1?

Reply:
Nope, v2 will suffice, updated.